src/index.html

```diff
@@ -48,19 +48,19 @@
 		<nav class="navigation__nav" role="navigation">
 			<ul class="navigation__list">
 				<li class="navigation__item">
-					<a href="#about" class="navigation__link"><span>01</span>About</a>
+					<a href="#about" class="navigation__link"><span>About</span></a>
```

Comment:
```suggestion
					<a href="#about" class="navigation__link">About</a>
```

Many Thanks @Zemerik for the PR, Can you also remove the span wrapping the text as it is no longer needed ?